QRServer/common/utils.py

```diff
@@ -0,0 +1,2 @@
+def is_guest(username: str):
+    return username.endswith(' GUEST')
```

Comment:
original game checks this by also comparing password hash to hash('<NOPASSWD>') but I'm not certain whether that is necessary.

Reply:
sure, I've added it